site/content/docs/0.0/getting-started/accessibility.md

```diff
@@ -8,4 +8,100 @@ aliases:
 toc: true
 ---
 
-{{< callout-soon "page" >}}
+OUDS Web provides an easy-to-use framework of ready-made styles, layout tools, and interactive components, allowing developers to create websites and applications that are visually appealing, functionally rich, and accessible out of the box.
+
+## Overview and limitations
+
+The overall accessibility of any project built with OUDS Web depends in large part on the author's markup, additional styling, and scripting they've included. However, provided that these have been implemented correctly, it should be perfectly possible to create websites and applications with OUDS Web that fulfill [<abbr title="Web Content Accessibility Guidelines">WCAG</abbr> 2.2](https://www.w3.org/TR/WCAG/) (A/AA/AAA), [Section 508](https://www.section508.gov/), and similar accessibility standards and requirements.
```

Comment:
WCAG 2.1 → WCAG 2.2 modification should be done in Bootstrap first. And globally if necessary.

Reply:
Yep, I'll do it Bs side first, I'll revert it in here.